packages/application-components/src/components/tabular-detail-page/tabular-detail-page.tsx

```diff
@@ -0,0 +1,82 @@
+import type { ReactElement, ReactNode, MouseEvent, KeyboardEvent } from 'react';
+import Spacings from '@commercetools-uikit/spacings';
+import { sharedMessages } from '@commercetools-frontend/i18n';
+import PageHeaderTitle from '../internals/page-header-title';
+import PageTopBar from '../internals/page-top-bar';
+import {
+  ControlsContainter,
+  TabularPageContainer,
+} from '../internals/tabular-page';
+import { ContentWrapper, PageWrapper } from '../internals/page.styles';
+
+// NOTE: the `MessageDescriptor` type is exposed by `react-intl`.
+// However, we need to explicitly define this otherwise the prop-types babel plugin
+// does not recognize the object shape.
+type MessageDescriptor = {
+  id: string;
+  description?: string | object;
+  defaultMessage?: string;
+};
```

Comment:
I think we should also add the `values` property to allow for messages that contain parameters as we had to do in ui-kit.

```suggestion
type MessageDescriptor = {
  id: string;
  description?: string | object;
  defaultMessage?: string;
  values?; Record<string, React.ReactNode>
};
```